libr/io/p/io_winedbg.c

```diff
@@ -236,7 +236,7 @@ static char *__system(RIO *io, RIODesc *fd, const char *cmd) {
 		eprintf ("pid : show current process id\n");
 	} else if (!strncmp (cmd, "dr8", 3)) {
 		struct winedbg_x86_32 r = regState ();
-		ut8 *arena = (ut8*)calloc (sizeof (struct winedbg_x86_32), 3);
+		ut8 *arena = (ut8*)calloc (3, sizeof (struct winedbg_x86_32));
```

Comment:
that doesnt changes the behaviour

Reply:
It does not :-) was just to silent a clang analysis warning.